src/router/Routes.tsx

```diff
@@ -4,12 +4,14 @@ import Home from '../pages/Home/Home';
 import NotFound from '../pages/NotFound/NotFound';
 import History from './history';
 import Login from '../pages/Login/Login';
+import {PublicModule} from '../pages/public-module/Public-module';
 
 const Routes = () => (
   <Router history={History}>
     <Switch>
       <Route exact={true} path='/' component={Home}/>
       <Route exact={true} path='/Login' component={Login}/>
+      <Route exact={true} path='/Public-module' component={PublicModule}/>
```

Comment:
Rename this to `/explore`. Please don't use uppercase in the url, can you adjust the login route to be lowercase as well?